src/app/create-long-post-page/create-long-post/create-long-post.component.ts

```diff
@@ -339,23 +333,19 @@ export class CreateLongPostComponent implements AfterViewInit {
       const permalink = `${window.location.origin}/u/${currentUserProfile.Username}/blog/${titleSlug}`;
       const postTx = await this.backendApi
         .SubmitPost(
-          this.globalVars.localNode,
           this.globalVars.loggedInUser?.PublicKeyBase58Check,
           this.editPostHashHex ?? "" /*PostHashHexToModify*/,
           "" /*ParentPostHashHex*/,
-          "" /*Title*/,
           {
             Body: `${postExtraData.Title}\n\n${postExtraData.Description}\n\nView this post at ${permalink}${
               entities ? `\n\nMentions: ${entities}` : ""
             }\n\n#blog`,
             ImageURLs: postExtraData.CoverImage ? [postExtraData.CoverImage] : [],
+            VideoURLs: null,
           } /*BodyObj*/,
           "" /*RepostedPostHashHex*/,
           postExtraData /*PostExtraData*/,
-          "" /*Sub*/,
-          false /*IsHidden*/,
-          this.globalVars.defaultFeeRateNanosPerKB /*MinFeeRateNanosPerKB*/,
```

Comment:
seems like the feeRateNanos is removed here as well. Is that ok?

Reply:
This should be fine bc the library uses the default value we pass to configure for transactions.